pepper-apis/src/main/java/org/broadinstitute/ddp/util/RouteUtil.java

```diff
@@ -223,6 +228,20 @@ public static void haltIfError(User user, String participantGuid, String studyGu
         }
     }
 
+    public static void haltError(int status, String code, String msg) {
+        LOG.warn(msg);
```

Comment:
One drawback of abstracting out a method like this is that this `LOG` object now refers to the `RouteUtil` class and not `AdminParticipantsLookupRoute`. So you won't be able to tell where the log entry came from in the logs.

Reply:
Fixed this: added method 'haltError' to a Route-class.